libr/anal/fcn.c

```diff
@@ -368,6 +368,23 @@ static const char *retpoline_reg(RAnal *anal, ut64 addr) {
 		const char *thunk = strstr (flag->name, token);
 		if (thunk) {
 			return thunk + strlen (token);
+		} else {
+			const char *tokens[] = {
```

Comment:
could you just add `x86_indirect_thunk_` to this list and just do this instead of having a special case for `x86_indirect_thunk_`?